### PR TITLE
Remove kyma 1 support

### PR DESCRIPTION
**Pull request by piotrmiskiewicz**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

### CODE REVIEW

File: components/kyma-environment-broker/cmd/broker/broker_suite_test.go [hunk 1]
```diff
@@ -886,7 +913,7 @@ func (s *BrokerSuiteTest) processProvisioningByOperationID(opID string) {
 	require.NoError(s.t, err)
 
 	// provisioner finishes the operation
-	s.WaitForOperationState(opID, domain.Succeeded)
+	//s.WaitForOperationState(opID, domain.Succeeded)
```
Comment (wozniakjan) [inline components/kyma-environment-broker/cmd/broker/broker_suite_test.go:916]: nit: intended to keep this commented out?

**Review by wozniakjan — COMMENTED** [review]
lgtm but looks like it conflicts with some recently merged PR

**Comment by szwedm** [comment]
New PR will be created to keep compatibility with the changes since this PR has been created.
Related issue: #1556

**Pull request closed without merging**